src/builderUtils/converters.ts

```diff
@@ -123,11 +123,11 @@ export const schema2value = (
     } else if (isArraySchema(schema)) {
       isArray = true;
       value = schema2value(schema.items);
-    } else if (schema.properties || schema.additionalProperties) {
+    } else if (schema.type === 'object' || schema.properties || 'additionalProperties' in schema) {
```

Comment:
```suggestion
    } else if (schema.type === 'object' || schema.properties || schema.additionalProperties) {
```
@mashabow PRの確認遅くなっておりすみません。こちら他の箇所と同様に `schema.additionalProperties`とするのはいかがでしょうか？